packages/client/src/products/__tests__/getProductOutfits.test.ts

```diff
@@ -0,0 +1,38 @@
+import { getProductOutfits } from '..';
+import { mockProductId, mockProductOutfits } from 'tests/__fixtures__/products';
+import client from '../../helpers/client';
+import fixtures from '../__fixtures__/getProductOutfits.fixtures';
+import mswServer from '../../../tests/mswServer';
+
+describe('getProductOutfits', () => {
+  const expectedConfig = undefined;
+  const spy = jest.spyOn(client, 'get');
+
+  beforeEach(jest.clearAllMocks);
+
+  it('should handle a client request successfully', async () => {
+    mswServer.use(fixtures.success(mockProductOutfits));
+
+    expect.assertions(2);
```

Comment:
Since you are awaiting the `resolves` promise below, you can remove this assertion as if the request fails, the `await` will also fail.

Reply:
Sorry, this was a mistake. Fixed